crates/cli-support/src/js/mod.rs

```diff
@@ -56,9 +56,17 @@ impl<'a> Context<'a> {
         let contents = deindent(contents);
         let contents = contents.trim();
         let global = if self.config.nodejs {
-            format!("module.exports.{} = {};\n", name, contents)
+            if contents.starts_with("class") {
+                format!("let {0} = null;\nmodule.exports.{0} = {0} = {1};\n", name, contents)
```

Comment:
To avoid the null could this perhaps instead be `let {0} = {1};\nmodule.exports.{0} = {0};\n`?

Reply:
Even simpler solution: "{1}\nmodule.exports.{0} = {0};\n" (it seems to work perfectly)